go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -49,11 +49,24 @@ func (api *EthereumAPI) GetBalance(_ context.Context, encryptedParams common.Enc
 
 // GetBlockByNumber returns the rollup with the given height as a block. No transactions are included.
 func (api *EthereumAPI) GetBlockByNumber(_ context.Context, number rpc.BlockNumber, _ bool) (map[string]interface{}, error) {
+	var rollupHeaderWithHashes *common.HeaderWithTxHashes
```

Comment:
Nitpick - don't think you need to predeclare this.

Reply:
Good spot, fixed.